<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>47</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="5">
    <widget class="QCheckBox" name="b3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QCheckBox" name="b2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="regName">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="maxVisibleItems">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QCheckBox" name="b1">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QCheckBox" name="b0">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="b5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="valueLabel">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="b7">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="b4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="b6">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="l7">
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="l6">
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="l5">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="l4">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="l3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="l2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLabel" name="l1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QLabel" name="l0">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>READ</string>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/control/reset.svg</normaloff>:/control/reset.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Frame</receiver>
   <slot>execute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>execute()</slot>
 </slots>
</ui>
